en/ros/ros2_comm.md

```diff
@@ -7,6 +7,7 @@ This topic provides an overview of the architecture and application pipeline, an
 :::note
 The [uXRCE-DDS](../middleware/uxrce_dds.md) middleware is supported in releases from **PX4 v1.14**.
 **PX4 v1.13** does not support ROS 2 via [uXRCE-DDS](../middleware/uxrce_dds.md) middleware (see [PX4 v1.13 Docs](https://docs.px4.io/v1.13/en/ros/ros2_comm.html) for information).
+If you were using Fast-RTPS and you now want to move to uXRCE-DDS, follow the [migration guide](../middleware/uxrce_dds.md#fast-rtps-to-uxrce-dds-migration-guidelines).
```

Comment:
Your change is good. However the whole thing feels middleware specific on a ROS 2 page. At this point people don't even know what the middleware is. So perhaps a note that is more PX4 and ROS 2 centric?

```suggestion
From PX4 v1.14, ROS 2 uses [uXRCE-DDS](../middleware/uxrce_dds.md) middleware, replacing the _FastRTPS_ middleware that was used in version 1.13 (v1.13 does not support uXRCE-DDS).

The [migration guide](../middleware/uxrce_dds.md#fast-rtps-to-uxrce-dds-migration-guidelines) explains what you need to do in order to migrate ROS 2 apps from PX4 v1.13 to PX4 v1.14. 

If you're still working on PX4 v1.13, please follow the instructions in the [PX4 v1.13 Docs](https://docs.px4.io/v1.13/en/ros/ros2_comm.html).
```

Reply:
Thanks @hamishwillee !